### ISSUE TITLE
xml parse errors

### ISSUE THREAD

**Issue by w0** [alectrona/git4jamfpro#6]
On some scripts, when making changes or trying to upload new. I get an error similar to this. 

`-:-1.0: unterminated entity reference & echo "Uninstalling ATP" && /usr/local/McAfee/uninstall ATP`

This will cause the modified script to be empty in jamf pro. Here is the full line in question

`[[ -f "/usr/local/McAfee/uninstall" ]] && echo "Uninstalling ATP" && /usr/local/McAfee/uninstall ATP`

It seems to be unhappy with my usage of &&? 

This line in another script tripped it up, with the following error. `-:18.34: xmlParseEntityRef: no name /usr/bin/caffeinate -d -i -m -u & caffeinatePID=$!`. This failed at script creation.

```# Caffeinate Mac to keep awake
/usr/bin/caffeinate -d -i -m -u & caffeinatePID=$!
```

**Comment by ryangball** [comment]
Hey @w0, try to pull the latest version. What I did in  #7 should resolve the issue.

**Comment by w0** [comment]
Got a new error this time!

```Determining changes between the last two git commits...
cat: '<?xml version="1.0" encoding="UTF-8"?>'$'\n''<script>'$'\n''  <name>Deployment</name>'$'\n''  <category>Scripts</category>'$'\n''  <info>Installs software before deployment</info>'$'\n''  <notes/>'$'\n''  <priority>After</priority>'$'\n''  <parameters>'$'\n''    <parameter4>Deployment Name</parameter4>'$'\n''  </parameters>'$'\n''  <os_requirements/>'$'\n''</script>': No such file or directory
-:1.1: Document is empty
^
```

**Comment by AllPurposeBen** [comment]
Getting the same:

```
Determining changes between the last two git commits...

cat: '<?xml version="1.0" encoding="UTF-8"?>'$'\n''<script>'$'\n''  <name>FireEye Uninstall</name>'$'\n''  <category>Security</category>'$'\n''  <info>This script will uninstall the FireEye client.</info>'$'\n''  <notes/>'$'\n''  <priority>After</priority>'$'\n''  <parameters/>'$'\n''  <os_requirements/>'$'\n''</script>': No such file or directory
-:1.1: Document is empty

^
Could not determine name of script from the xml record, skipping.
```

That tracks back to [line 87](https://github.com/alectrona/git4jamfpro/blob/9a7c9683b9c33c6dc5374787fe9f32a7f83102a0/git4jamfpro.sh#L87) and I think this issue might be a `cat` where an `echo` should be on that line. Fixed it on my local and is working, I'll update my fork and submit a PR for the fix here shortly.

**Comment by ryangball** [comment]
This should be fixed with #9 and #10.
